Source/WebKit/UIProcess/Cocoa/CoreTelephonyUtilities.mm

```diff
@@ -77,7 +77,11 @@ bool shouldAllowAutoFillForCellularIdentifiers(const URL& topURL)
         return false;
     }
 
+#if HAVE(DELAY_INIT_LINKING)
+    static NeverDestroyed cachedClient = adoptNS([[CoreTelephonyClient alloc] initWithQueue:dispatch_get_main_queue()]);
```

Comment:
If CoreTelephony.framework is missing from an SDK, this is now a build-time failure. If it's missing at runtime, it's a launch failure. Previously it would have built regardless, and only crashed in this function (which may not be compiled at all due to the HAVE guard). Is that safe?

Reply:
> Checking on SDKs and images.

Running some internal commands to check SDKs and images, I don't think the `CoreTelephonyClient` Objective-C class will cause any build-time or launch-time issues.

I also checked with a manager on the team, and they weren't aware of any places where `CoreTelephony.framework` might not be available.

Reply:
Add @whsieh to the reviewers list in case he has anything to add about the "intent" when soft-linking `CoreTelephony.framework`.